test/k8s-integration/driver.go

```diff
@@ -71,9 +72,9 @@ func installDriver(goPath, pkgDir, stagingImage, stagingVersion, deployOverlayNa
 	statusCmd := exec.Command("kubectl", "describe", "pods", "-n", driverNamespace)
 	err = runCommand("Checking driver pods", statusCmd)
 	if err != nil {
-		return fmt.Errorf("failed to check driver pods: %v", err)
+		return fmt.Errorf("failed to get nodes: %v", err)
 	}
-
+	klog.V(2).Infof("GCE PD driver pods: \n %v", string(out))
```

Comment:
`out` is no longer relevant in this case

Reply:
fixed. PTAL